Libs/MRML/Widgets/qMRMLLayoutManager.cxx

```diff
@@ -960,6 +960,13 @@ qMRMLThreeDWidget* qMRMLLayoutManager::threeDWidget(int id)const
     this->mrmlViewFactory("vtkMRMLViewNode")->viewWidget(id));
 }
 
+//------------------------------------------------------------------------------
+qMRMLThreeDWidget* qMRMLLayoutManager::threeDWidget(const QString& name)const
+{
+  return qobject_cast<qMRMLThreeDWidget*>(
+    this->mrmlViewFactory("vtkMRMLViewNode")->viewWidget(name));
```

Comment:
Check for null pointer

Reply:
OK I'll add that to the other similar functions too then.